app/web/template/dashboard.gohtml

```diff
@@ -42,7 +42,11 @@
               <h2 class="govuk-heading-m govuk-!-padding-top-0 govuk-!-margin-bottom-1">{{ if eq "pfa" .Type }}{{ tr $.App "lpaTypePfa" }}{{ else }}{{ tr $.App "lpaTypeHw" }}{{ end }}: <span class="govuk-!-font-weight-regular">{{ .You.FirstNames }} {{ .You.LastName }}</span></h2>
               <span class="govuk-hint"><strong>{{ tr $.App "applicationNumber" }}:</strong> {{ .ID }}</span>
               <div class="govuk-button-group govuk-!-margin-top-4">
-                <a class="govuk-button" href="{{ link $.App (printf "%s%s" .ID $.App.Paths.TaskList) }}">{{ tr $.App "continue" }}</a>
+                {{ if eq .Progress.LpaSigned.String "completed" }}
```

Comment:
I added `.Completed` and `.InProgress` methods to `TaskState` to solve this

Reply:
My bad, I'd forgotten to update this after I'd used the funcs on the progress partial.